e2e/specs/st_multiselect.spec.ts

```diff
@@ -36,10 +36,10 @@ describe("st.multiselect", () => {
 
     it("should show the correct text", () => {
       cy.get(".stText")
-        .should("have.length", 4)
+        .should("have.length", 5)
         .should(
           "have.text",
-          "value 1: []value 2: []value 3: []value 4: ['tea', 'water']"
+          "value 1: []value 2: []value 3: []value 4: ['tea', 'water']value 5: []"
```

Comment:
Is the spacing on this correct?

Reply:
Believe so. If you're referring to the lack of spacing between `[]value`, that's how cypress reads the string if there are multiple elements. 
```should('have.text' checks the full combined text of all matched elements.```

Reply:
:+1: